packages/boot/test/bootstrapTests/test-vat-orchestration.ts

```diff
@@ -9,6 +15,16 @@ const makeTestContext = async (t: ExecutionContext) =>
 type DefaultTestContext = Awaited<ReturnType<typeof makeTestContext>>;
 const test: TestFn<DefaultTestContext> = anyTest;
 
+// marked success in [support.ts](../tools/support.ts) based on the packets `data` key
```

Comment:
These are two sentences but all lowercase and no punctuation makes that slow to parse

Reply:
Updated here: https://github.com/Agoric/agoric-sdk/pull/9195/commits/e49a8376e69187ea40fc9d96c9ce517d9cfe2897